Intake/Onboarding/Welcome.swift

```diff
@@ -16,35 +16,35 @@ struct Welcome: View {
     
     var body: some View {
```

Comment:
Great work on updating the texts in the onboarding flow, I like the used SF symbols a lot!

Please be sure that you use localized strings from the Strings catalog. You can learn more about localization here and I would highly recommend watching the video to completely understand the localization process that we use in Spezi-based applications: https://developer.apple.com/wwdc23/10155